other-products/subbridge/technical-details.md

```diff
@@ -53,19 +53,22 @@ The registration of SubBridge assets is mainly divided into two parts:
 
 The first part is to register assets into the pallet-assets module. SubBridge uses the pallet-assets module provided by Substrate to manage the registered assets. The registered assets will be assigned an asset id. Each asset has an extra [registry info](https://github.com/Phala-Network/khala-parachain/blob/5ab4f77163c811fb4a02d337791ce669b41481ad/pallets/assets-registry/src/lib.rs#L62) which contains information of location, enabled bridges and properties. Unregistered assets will fail regardless of whether they are transferred via the EVM bridge or the XCM bridge.
 
-The second part is to enable the corresponding EVM bridge. This part is only for the asset settings that want to carry out the cross-chain requirement from Khala to the EVM chain. In SubBridge, the same asset can enable both ChainBridge-based bridges and CelerBridge-based bridges (coming soon). In practice, users are always willing to choose solutions with lower fees.
+The second part is to enable the corresponding EVM bridge. This part is only for the asset settings that want to carry out the cross-chain requirement from Phala to the EVM chain. In SubBridge, the same asset can enable both SygmaBridge-based bridges and ChainBridge-based bridges. In practice, users are always willing to choose solutions with lower fees.
 
 The steps to do the registration stuff are as follow:
 
-*   Step1, we schedule a call of `pallet-registry::forceRegisterAsset` with given registration informations. When the council enacted the call, an asset instance will be created by `pallet-assets`, and some extra registration information will be saved in `pallet-registry`.
+*   Step1, we schedule a call of `pallet-registry::forceRegisterAsset` with given registration informations. When the technical committee enacted the call, an asset instance will be created by `pallet-assets`, and some extra registration information will be saved in `pallet-registry`.
```

Comment:
Are you sure it's the technical committee?

Reply:
Yeah, it's Council, fixed already